enhancements/cluster-logging/api-audit-log-policy.md

```diff
@@ -0,0 +1,254 @@
+
+#### Run-time resource costs
+
+OSD and ROSA classic have been doing this type of filtering on managed clusters for years, no new problems are expected.
+
+#### Compliance and legal risks
+
+Incorrectly filtering out important audit events may be a compliance/legal problem.
+The customer creates their own filtering configuration, it needs to be clear that it is their responsibility to do it correctly.
+
+#### Roll-out
+
+Existing customers using the CLO add-on will be unaffected.
+The customer `cl/instance` and `clf/instance` resources will not be modified.
+The new features will be activated only when the customer modifies their configuration as described above.
+
+The existing SRE Splunk exporter SHOULD be replaced by separate CL/CLF resources in a separate namespace as soon as possible.
+Depends on [LOG-1343 Multiple forwarders](https://issues.redhat.com/browse/LOG-1343).
+
+However, the Splunk exporter MAY continue to run alongside customer CLO deployments using these new features, if necessary.
+Both can scrape from the same audit log files simultaneously, and perform separate filtering.
+
+**NOTE**: this is not true for Hypershift, [see hypershift proposal](hypershift-audit-logs.md)
+
+### Drawbacks
+
+- Increased scope for user error.
+- Additional CPU cost, latency (to be measured)
+- New dependency to be maintained for the log collector.
+
+## Design Details
+
+### Open Questions
+
+#### Form of audit policy
+
+The [Kubernetes Audit Policy File](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/) is expressed as a k8s "object" but not as a typical k8s "resource (no spec/status).
+
+Should we
+- Use it as a resource unmodified
+- Wrap it in a more typical spec/status resource
+
+### Test Plan
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+#### Tech Preview -> GA
+
+#### Removing a deprecated feature
+
+None.
+
+### Upgrade / Downgrade Strategy
+
+Nothing new.
+
+### Version Skew Strategy
+
+The CLO's installed on management and hosted clusters may be different versions.
+
+The CLO on the _management_ cluster forwards
+- All logs required by the provider, configured by `ClusterLogForwarderTemplate` on _management_ cluster
+- API-audit logs _only_ for the customer, configured by `HypershiftLogForwarder` on _hosted_ cluster
+
+The CLO on the _hosted_ cluster forwards
+- All non-API-audit logs, configured by `ClusterLogForwarder` on the _hosted_ cluster.
+
+If the hosted/managent CLO versions are different, the customer can
+- Create `ClusterLogForwarder` based on the _hosted_ CLO version.
+- Create `HypershiftLogForwarder` based on the _management_ CLO version.
+
+This might be confusing but there is a clear separation of resources associated with each version.
+- There's no situation where the customer is blocked from upgrading their hosted CLO.
+- A customer could be blocked from sending API-audit logs (only) to a new type of log store if the SD CLO is old.
```

Comment:
Contents of this section should be merged into hypershift-audit-logs.md.  Not sure what would be appropriate here in api-audit-log-policy.md.